### PR TITLE
Dfr 1092 update opening hours

### PR DESCRIPTION
**Pull request by BWeeks-JusticeGovUK**

### CODE REVIEW

File: k8s/namespaces/divorce/div-da/demo.yaml [hunk 1]
```diff
@@ -6,7 +6,6 @@ spec:
   releaseName: div-da
   values:
     nodejs:
-      image: hmctspublic.azurecr.io/div/dn:pr-252-2fa3313-20220810085324 #{"$imagepolicy": "flux-system:demo-div-da"}
```
Comment (thomast1906) [inline k8s/namespaces/divorce/div-da/demo.yaml:9]: you need this or it will use the base `image`

File: k8s/namespaces/divorce/div-pfe/demo.yaml [hunk 2]
```diff
@@ -6,7 +6,7 @@ spec:
   releaseName: div-pfe
   values:
     nodejs:
-      image: hmctspublic.azurecr.io/div/pfe:pr-1027-f81b74a-20220810131202 #{"$imagepolicy": "flux-system:demo-div-pfe"}
+      image: hmctspublic.azurecr.io/div/pfe:prod-0cdf4ab-20220725164638 #{"$imagepolicy": "flux-system:div-pfe"}
```
Comment (thomast1906) [inline k8s/namespaces/divorce/div-pfe/demo.yaml:9]:
```suggestion
      image: hmctspublic.azurecr.io/div/pfe:prod-0cdf4ab-20220725164638 #{"$imagepolicy": "flux-system:demo-div-pfe"}
```

File: k8s/namespaces/divorce/div-rfe/demo.yaml [hunk 3]
```diff
@@ -5,8 +5,8 @@ metadata:
 spec:
   releaseName: div-rfe
   values:
-    nodejs:
-      image: hmctspublic.azurecr.io/div/rfe:prod-1afcef2-20220801115254 #{"$imagepolicy": "flux-system:demo-div-rfe"}
+    nodejs:      
+      image: hmctspublic.azurecr.io/div/rfe:prod-1afcef2-20220801115254 #{"$imagepolicy": "flux-system:div-rfe"}
```
Comment (thomast1906) [inline k8s/namespaces/divorce/div-rfe/demo.yaml:9]:
```suggestion
      image: hmctspublic.azurecr.io/div/rfe:prod-1afcef2-20220801115254 #{"$imagepolicy": "flux-system:demo-div-rfe"}
```

File: k8s/namespaces/divorce/div-da/demo.yaml (hunk 1 above)
Comment (BWeeks-JusticeGovUK) [reply k8s/namespaces/divorce/div-da/demo.yaml:9]: fixed

**Review by thomast1906 — APPROVED** [review]

**Review by Michael1142 — APPROVED** [review]

**Pull request merged**